scripts/transform_zipcodes.rb

```diff
@@ -0,0 +1,16 @@
+require 'csv'
```

Comment:
Also, these large files don't really need to be in version control but for now they sort of do.